crates/client/src/config.rs

```diff
@@ -122,6 +122,10 @@ pub struct Config {
     /// Auto accept registry hint or ask the user to confirm
     #[serde(default)]
     pub auto_accept_federation_hints: bool,
+
+    /// Disable dialoguer prompts.
+    #[serde(default)]
+    pub disable_dialoguer: bool,
```

Comment:
I wouldn't reference the specific library here. Maybe `disable_interaction`?

Reply:
Fixed